content_script.js

```diff
@@ -0,0 +1,10 @@
+function handleMessage(request) {
+    pauseVideo() //we don't care about message contents, pause on any message received
```

Comment:
Not sure if there is need for any more messages so I left this without any message content checking.

Reply:
Thanks, that makes sense. Mind renaming the `request` parameter to `_request` to emphasize that we aren't currently using it?